Alignment/CommonAlignmentProducer/plugins/AlignmentProducer.cc

```diff
@@ -641,7 +641,7 @@ void AlignmentProducer::simpleMisalignment_(const Alignables &alivec, const std:
 
 
 //__________________________________________________________________________________________________
-void AlignmentProducer::createGeometries_( const edm::EventSetup& iSetup )
+void AlignmentProducer::createGeometries_( const edm::EventSetup& iSetup, const TrackerTopology* tTopo )
```

Comment:
TrackerTopology is a part of EventSetup. It would be better to retrieve it
from it directly when needed.

Yana

On Friday, January 22, 2016, Marco Musich <notifications@github.com> wrote:

> In Alignment/CommonAlignmentProducer/plugins/AlignmentProducer.cc
> <https://github.com/cms-sw/cmssw/pull/12995#discussion_r50545763>:
>
> > @@ -641,7 +641,7 @@ void AlignmentProducer::simpleMisalignment_(const Alignables &alivec, const std:
> >
> >
> >  //__________________________________________________________________________________________________
> > -void AlignmentProducer::createGeometries_( const edm::EventSetup& iSetup )
> > +void AlignmentProducer::createGeometries_( const edm::EventSetup& iSetup, const TrackerTopology* tTopo )
>
> @ianna <https://github.com/ianna> are you fine with this? this is the
> only change that is touching alca packages
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cms-sw/cmssw/pull/12995/files#r50545763>.
>